applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -104,13 +104,13 @@ class MapperVertexMorphing
     ///@{
 
     /// Default constructor.
-    MapperVertexMorphing( ModelPart& designSurface, Parameters optimizationSettings )
+    MapperVertexMorphing( ModelPart& designSurface, Parameters MapperSettings )
```

Comment:
"MapperSettings" should be "mapper_settings" 
(But maybe we shift this whole renaming thing to the restructuring of the optimization)

Reply:
done